src/DssSpell.sol

```diff
@@ -109,6 +122,46 @@ contract DssSpellAction is DssAction {
         // Increase the GNO-A Stability Fee from 2.50% to 4.90%
         DssExecLib.setIlkStabilityFee("GNO-A", FOUR_NINE_PCT_RATE, /* doDrip = */ true);
 
+        // ---------- PE MKR Vesting Stream Cleanup ----------
+
+        // PE-001 Contributor - 248 MKR - 0x18A0609b14dB84bbcC3d911915a07CA9a28b9263
+        GemLike(DssExecLib.mkr()).transfer(PE_CONTRIBUTOR, 248 ether);  // NOTE: 'ether' is a keyword helper, only MKR is transferred here
```

Comment:
Shouldn't we use `WAD` instead of `ether` here?

Reply:
Yeap, can do that.